source/services/dep-tracking.rst

```diff
@@ -0,0 +1,49 @@
+Dependency Tracking
+###################
+
+Projects rely heavily on often a large number of third-party packages, and those
+packages hopefully continue to receive updates, bug fixes, optimizations, and
+new features. We want to be aware of those updates when they are available,
+especially when new available updates include security fixes that would be
+particularly important for us to update in our projects.
+
+Requires.io
+===========
+
+Our Python dependencies can be tracked with the requires.io tool.
+
+There are two modes to integrate with, depending on if you're using a public repository or a
+private one. If you're using a public repository, you can log into requires.io using your GitHub
+account.
+
+You'll be presented with a list of available public repositories to navigate for all of your
+repositories. Navigate to your new project and click the "Activate" button. On the project details
+page choose the branch you'd like to show requirements for, preferably the master branch. You'll
+see a report of dependencies for this branch.
```

Comment:
Can we check both master and develop, so we can fix out-of-date things before promoting them to master?